src/sentry/options/defaults.py

```diff
@@ -43,3 +43,5 @@
 # symbolizer specifics
 register('dsym.llvm-symbolizer-path', type=String)
 register('dsym.cache-path', type=String, default='/tmp/sentry-dsym-cache')
+
+register('mail.list-namespace', type=String, default='localhost', flags=FLAG_NOSTORE)
```

Comment:
Oh I mostly mean just show what the value is -- we dont allow you to configure any of the mail settings in the UI yet (I dont think?)